app/src/main/res/values/dimens.xml

```diff
@@ -1,4 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <dimen name="dp_4">4dp</dimen>
+
+    <!-- Used for next commit -->
+    <!--<dimen name="dp_48">48dp</dimen>-->
+    <!--<dimen name="sp_11">11sp</dimen>-->
+    <!--<dimen name="dp_8">8dp</dimen>-->
+    <!--<dimen name="sp_24">24sp</dimen>-->
+
+    <dimen name="dp_16">16dp</dimen>
+    <dimen name="sp_32">32sp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.sp_32` appears to be unused
 Unused resources make applications larger and slow down builds.